ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -248,26 +248,6 @@ export const queryDirectory = (): Promise<boolean> =>
       });
   });
 
-function toUint8Array(array: Uint8Array) {
-  const buffer = new ArrayBuffer(array.length);
-  const newArray = new Uint8Array(buffer);
-  for (let i = 0; i < array.length; i += 1) {
-    newArray[i] = array[i];
-  }
-
-  return newArray;
-}
-
-// function getBytes(str: string) {
-//     let bytes: number[] = []; // char codes
-//     for (let i = 0; i < str.length; i+=1) {
-//       const code = str.charCodeAt(i);
-//       bytes = bytes.concat([code]);
-//     }
-
-//     return bytes;
-// }
-
 export const createAndDeleteFile = (): Promise<boolean> =>
   new Promise((resolve, reject) => {
```

Comment:
I didn't pick up on this before but it might be worth refactoring this to be an `async` function with `await` statements. Under the hood `async` functions return promises, and `await` waits for the promise to resolve.